app/hook/useDate.ts

```diff
@@ -78,17 +77,24 @@ export default function useDate() {
     const dataString = `${day} ${monthLetters}, ${year}`
     return dataString
   }
-
-  const generateEndpointCardPost = ({ date, title }: GenerateEndPointCardPostParam): string => {
+  // old generation endpoint by cardPost
```

Comment:
is possible to remove all these comments?

Reply:
Yes, ofcourse.